.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "eslint.nodePath": "./node_modules/eslint",
+  "[typescript]": {
+    "editor.codeActionsOnSave": {
+      "source.fixAll.eslint": true
+    }
+  },
+  "eslint.options": {
+    "overrideConfigFile": "./.eslintrc"
+  }
```

Comment:
Is this particular setting needed? eslint picks up setting automatically from the root of the repo.

And if someone didn't add the root of the repo to the VSCode window then this setting wouldn't apply anyway (I believe).